vsintegration/src/FSharp.LanguageService/QuickParse.fs

```diff
@@ -240,42 +241,42 @@ module internal QuickParse =
 
             and InUnquotedIdentifier(left:int,pos:int,current,throwAwayNext) =
                 if pos > index then 
-                    if throwAwayNext then [],"" else current,(line.Substring(left,pos-left))
+                    if throwAwayNext then defaultName else current,line.Substring(left,pos-left)
                 else
-                    if IsIdentifierPartCharacter(pos) then InUnquotedIdentifier(left,pos+1,current,throwAwayNext)
-                    elif IsDot(pos) then 
+                    if IsIdentifierPartCharacter pos then InUnquotedIdentifier(left,pos+1,current,throwAwayNext)
+                    elif IsDot pos then 
                         let ident = line.Substring(left,pos-left)
                         AtStartOfIdentifier(pos+1,ident::current,throwAwayNext)
-                    elif IsWhitespace(pos) || IsStartOfComment(pos) then 
+                    elif IsWhitespace pos || IsStartOfComment pos then 
                         let ident = line.Substring(left,pos-left)
                         SkipWhitespaceBeforeDotIdentifier(pos, ident, current,throwAwayNext)
                     else AtStartOfIdentifier(pos,[],false) // Throw away what we have and start over.
 
             and InQuotedIdentifier(left:int,pos:int, current,throwAwayNext) =
                 if pos > index then 
-                    if throwAwayNext then [],"" else current,(line.Substring(left,pos-left))
+                    if throwAwayNext then defaultName else current,line.Substring(left,pos-left)
                 else
-                    let remainingLength = line.Length-pos
-                    if IsTick(pos) && remainingLength>1 && IsTick(pos+1) then 
+                    let remainingLength = line.Length - pos
```

Comment:
@forki - You haven't really added spaces around other arithmetic operators ... is this intentional?